src/applications/vaos/appointment-list/components/PageLayout.jsx

```diff
@@ -3,16 +3,22 @@ import classNames from 'classnames';
 import PropTypes from 'prop-types';
 import { MhvSecondaryNav } from '@department-of-veterans-affairs/mhv/exports';
 import { useLocation } from 'react-router-dom';
+import { useSelector } from 'react-redux';
 import Breadcrumbs from '../../components/Breadcrumbs';
 import NeedHelp from '../../components/NeedHelp';
+import { selectFeatureListViewClinicInfo } from '../../redux/selectors';
 
+const listPageRegex = /^\/(past\/?)?$/; // matches '/' (this is the appointments list root route), '/past' or '/past/' -- not pending or referrals-requests
```

Comment:
`/pending` uses `PageLayout.jsx`. `/referrals-requests` uses `ReferralLayout.jsx` Wen you include `/pending`, the internals of that component still use the 8 column width even if you expand the layout to 10... so not including it seems appropriate especially since we are making no changes to the width of referrals-requests (same list content).